themes/src/main/resources/theme/base/admin/resources/js/controllers/realm.js

```diff
@@ -104,7 +104,7 @@ module.controller('GlobalCtrl', function($scope, $http, Auth, Current, $location
         if(Current.realm !== null && currentRealm !== Current.realm.id) {
             currentRealm = Current.realm.id;
             translateProvider.translations(locale, resourceBundle);
-            RealmSpecificLocalizationTexts.get({id: currentRealm, locale: locale}, function (localizationTexts) {
+            RealmSpecificLocalizationTexts.get({id: Current.realm.realm, locale: locale}, function (localizationTexts) {
```

Comment:
This is confusing. Either the API is using realm `id` - then the original code is correct, or the API is using unique realm name (`realm`) - then this fix should be accompanied with the change in `RealmSpecificLocalizationTexts` that renames `id` to `realm` for the sake of consistency.

Reply:
services.js contains 141x the line "realm : '@realm'" and 7x the line "id : '@realm'". I agree that should be fixed, but I am not confident enough to do that as backend dev without angular knowledge :-)

Reply:
I don't think there is anything to change. Otherwise, we would be asking for refactoring (IMO, not really the scope here) other parts of the admin console too.

From my side, looks good.

Reply:
Yup, this should have been caught earlier, when introducing the first inconsistency. I don't see this as a blocinkg issue, rather a reminder that we should be more careful when doing PR reviews.

Reply:
@hmlnarik True, thanks Hynek. Hope we improve test coverage of the admin console. Something I think is being considered from day one on the new account console. Mainly specific use cases like this one (admin permissioning).